### PR TITLE
Include any targets from the opened directory in the tree view.

### LINKED ISSUE #46: Targets at the top level of a repository don't appear in the tree view
We're not even querying the targets here.

### PR DESCRIPTION
**Pull request by thomasvl**
Fixes #46

### CODE REVIEW

File: src/workspace-tree/bazel_workspace_folder_tree_item.ts [hunk 1]
```diff
@@ -141,7 +143,7 @@ export class BazelWorkspaceFolderTreeItem implements IBazelTreeItem {
   /**
    * Returns a promise for an array of tree items representing build packages.
    */
-  private async getPackages(): Promise<BazelPackageTreeItem[]> {
+  private async getPackages(): Promise<IBazelTreeItem[]> {
```
Comment (allevato) [inline src/workspace-tree/bazel_workspace_folder_tree_item.ts]: Nit: for future us, we should probably change the name of this method now to `getTopLevelItems` (and update the call site in `getChildren`) so it's more accurate.

Comment (thomasvl) [reply src/workspace-tree/bazel_workspace_folder_tree_item.ts]: `getTopLevelItems` isn't really right either, _top_ implies (to me aleast) things at the WORKSPACE level.  This is getting the packages and items from the opened directory.

Comment (allevato) [reply src/workspace-tree/bazel_workspace_folder_tree_item.ts]: Well, the function is kind of odd in the sense that it gets *all* the packages but then builds the whole tree and only returns the top-level items.

`buildPackageTreeAndReturnTopLevelPackagesAndTargets`? 😅

Comment (thomasvl) [reply src/workspace-tree/bazel_workspace_folder_tree_item.ts]: That better?

Comment (allevato) [reply src/workspace-tree/bazel_workspace_folder_tree_item.ts]: WFM

**Pull request merged**